Don't run /sbin/chkconfig after install data. 
chkconfig is used to manage system services in redhat series linux distribution. 
It don't fit to Debian. 
--- a/Makefile.am
+++ b/Makefile.am
@@ -95,13 +95,11 @@
 	$(INSTALL_SCRIPT) lldpad.init $(DESTDIR)/etc/init.d/lldpad
 
 install-data-hook:
-	/sbin/chkconfig --add lldpad || true
 	## provide legacy support for apps that use the old dcbd interface.
 	test -e $(includedir)/dcbd || `ln -T -s $(includedir)/lldpad $(includedir)/dcbd`
 	test -e $(includedir)/dcbd/clif_cmds.h || `ln -T -s $(includedir)/lldpad/lldp_dcbx_cmds.h $(includedir)/dcbd/clif_cmds.h`
 
 uninstall-local:
-	/sbin/chkconfig --del lldpad || true
 	rm -f '$(DESTDIR)/etc/init.d/lldpad'
 	rm -f '$(includedir)/dcbd/clif_cmds.h'
 	rm -f '$(includedir)/dcbd'
